includes/pages/admin_user.php

```diff
@@ -188,8 +189,14 @@ function admin_user()
             . '</td></tr>' . "\n";
 
         $html .= '</table>' . "\n" . '<br>' . "\n";
-        $html .= '<button type="submit" class="btn btn-primary">'
-            . icon('save') . __('form.save') . '</button>' . "\n";
+        $html .= buttons([
+            '<button type="submit" class="btn btn-primary">' . icon('save') . __('form.save') . '</button>',
+            button(
+                url('/admin-user', ['action' => 'reset_pw', 'id' => $user_id]),
+                icon('envelope-at') . __('password.recovery.send'),
```

Comment:
Technically this is a legacy text and thus not dotted (and i am not sure if the dotted variation here fits that well, eventually some form of settings.password_recovery.send maybe would fit more? (or just ignore it and use the English text to "fit" in the page ;) )